src/sentry/models/groupassignee.py

```diff
@@ -141,6 +187,33 @@ def deassign(self, group: Group, acting_user: User | RpcUser | None = None) -> N
             GroupOwner.invalidate_assignee_exists_cache(group.project.id, group.id)
             GroupOwner.invalidate_debounce_issue_owners_evaluation_cache(group.project.id, group.id)
 
+            if (
+                features.has("organizations:participants-purge", group.organization)
+                and previous_assignee
+            ):
+                if (
+                    features.has("organizations:team-workflow-notifications", group.organization)
+                    and type(previous_assignee) is Team
+                ):
+                    GroupSubscription.objects.filter(
```

Comment:
seems like we don't have a test for this block?

Reply:
added one